.golangci-lint.yml

```diff
@@ -80,11 +80,18 @@ linters:
     - gosimple
     - staticcheck
     - unused
+    - tagalign
+    - wrapcheck
+    - errorlint
+    - wastedassign
 
 linters-settings:
   govet:
     enable:
       - fieldalignment
+  tagalign:
+    align: false
+    sort: false
```

Comment:
I would change these to true

Reply:
![image](https://github.com/bitcoin-sv/spv-wallet/assets/155538368/7d5619e8-ede5-416c-bad6-37c4dc34940a)
If we turn on align we get something like this. In places where comments separate lines of code, the formatting is different than in places where there are no separators

Reply:
I've found the same issue in other package. If you remove comments first and then align them it will be correct structure. After that you can add comments

Reply:
tagalign linter removed